src/index.html

```diff
@@ -10,7 +11,36 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <!-- #endregion -->
+  <body class="body">
+    <form
+      action="/"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
+      <label for="input-big" class="icon icon--big"></label>
+      <input
```

Comment:
Move the input element inside the label element to ensure better accessibility and maintain a cleaner HTML structure.